pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -241,3 +258,113 @@ def cmd(self):
 
         cmd = (" ").join(cmd)
         return cmd
+
+
+class Pa11yCrawler(BokChoyTestSuite):
+    """
+    Sets up test environment with mega-course loaded, and runs pa11ycralwer
+    against it.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(Pa11yCrawler, self).__init__(*args, **kwargs)
+        self.course_key = kwargs.get('course_key', "course-v1:edX+Test101+course")
+        if self.imports_dir:
```

Comment:
@cpennington  Instead of using a checksum for determining whether or not a course should be fetched, I've included this logic to allow the user to specify whether or not it should be.  `--skip-fetch` and `--fasttest` will both skip fetching and unarchiving the course.  If `--imports_dir` is specified by the user, it will skip the fetch since the usage of the `--imports_dir` flag assumes that you are pointing it to an unarchived course.